charts/tfy-agent/values.yaml

```diff
@@ -1,123 +1,296 @@
-# Default values for sample.
+# Default values for tfy-agent.
 # This is a YAML-formatted file.
 # Declare variables to be passed into your templates.
 
-global: {}
+# global settings
+global:
+  rbac:
+    enabled: true
 
-image:
-  repository: truefoundrycloud/tfy-agent
-  pullPolicy: IfNotPresent
-  # Overrides the image tag whose default is the chart appVersion.
-  tag: "a9af8a51d8b0373f4d0c469119ed88a488c5709f"
+# configuration for tfy-agent and tfy-agent-proxy
+config:
```

Comment:
Normally these are done via the `.env.default` approach.

https://github.com/truefoundry/tfy-agent/blob/main/.env.default
https://github.com/truefoundry/helm-charts/blob/fd9a78ec2a7fc3b21f18cc610f1eff6d591b478c/charts/tfy-agent/values.yaml#L58

Going forward, do we need to also add a config field, if we introduce an env var?

Reply:
We will introduce `extraEnvVars: []` feature for both applications in coming release.